src/styles/index.scss

```diff
@@ -2,2 +1,17 @@
+@use './header.scss';
+@use './main.scss';
+
+.page {
+  display: flex;
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+  scroll-behavior: smooth;
+}
+
+.body {
+  margin: 0;
+  width: 100%;
+  box-sizing: border-box;
 }
```

Comment:
The `.body` class also seems to be applied to the `body` or `html` element. As mentioned above, apply these styles directly to the `body` or `html` tag. Also, margin and box-sizing are already defined in the `.page` class. There is no need to duplicate them here.